website/docs/add-secure-apps/providers/oauth2/client_credentials.md

```diff
@@ -30,17 +30,40 @@ In addition to that, with authentik 2024.4 it is also possible to pass the confi
 
 ### JWT-authentication
 
-Starting with authentik 2022.4, you can authenticate and get a token using an existing JWT.
+#### Externally issued JWTs <span class="badge badge--version">authentik 2022.4+</span>
 
-(For readability we will refer to the JWT issued by the external issuer/platform as input JWT, and the resulting JWT from authentik as the output JWT)
+You can authenticate and get a token using an existing JWT. For readability we will refer to the JWT issued by the external issuer/platform as input JWT, and the resulting JWT from authentik as the output JWT.
 
-To configure this, the certificate used to sign the input JWT must be created in authentik. The certificate is enough, a private key is not required. Afterwards, configure the certificate in the OAuth2 provider settings under _Verification certificates_.
+To configure this, define a JWKS URL/raw JWKS data in OAuth Sources. If a JWKS URL is specified, authentik will fetch the data and store it in the source, and then select the source in the OAuth2 Provider that will be authenticated against.
```

Comment:
data source?  "...define a JWKS URL/raw JWKS data in OAuth Sources." is an incomplete sentence, needs the noun.